src/nvim/event/process.c

```diff
@@ -228,29 +228,26 @@ void process_stop(Process *proc) FUNC_ATTR_NONNULL_ALL
   }
 
   Loop *loop = proc->loop;
-  if (!loop->children_stop_requests++) {
```

Comment:
Small cleanup: I removed `loop->children_stop_requests` because it seems useless.  It will always be 0 here, because `process_stop()` is guarded by `proc->stopped_time`.